intro-to-git/activity-git-history.md

```diff
@@ -0,0 +1,93 @@
+# Activity: git History
+
+<!-- Can be async -->
+<!-- Written to be synchronous -->
+<!-- Ideal Format: Small groups, large groups, roundtables, etc. -->
+
+## Goal
+
+Our goal is to practice the process of making commits as a group. We will also practice collaborating in Git as a group.
+
+Our goal is to practice this without actually typing and running these commands in the terminal; however, we should practice recognizing what commands we _would_ type.
+
+## Activity Instructions
+
+1. Assemble into small groups
+1. Read through the instructions of Part 1
+1. Do Part 1
+1. Read through the instructions of Part 2
+1. Do Part 2
```

Comment:
Maybe remove this section? To me, it doesn't add anything, as students will likely read and do each part as they go.

Perhaps we could rephrase this as "Find a Partner (or Two): Find someone you haven't worked with before and pair up for this exercise"

Reply:
Coool! I'll do something with this. @cwboden @anselrognlie for an explanation on my thought process: Telling students to read through the instructions is _**definitely**_ a step that needs to be called out in the instructions, is what I've learned as a teacher, LOLOLOL.

Particularly since what I want for the git activity is "know what the goal is with git, THEN think about the steps to get there." IMO, git is mostly taught as "do this, then this, then this, then this, and then at some point you're done," and that's not how I necessarily want git to stick in their brains at first glance.